README.md

```diff
@@ -1,213 +1,49 @@
-## Model Description File Specification
+# Model Description File Specification
 
-This specification defines a file format for representing pretrained AI models in [YAML format](https://en.wikipedia.org/wiki/YAML). This format is used to describe models hosted in the [BioImage.IO](https://bioimage.io) model repository site.
-
-To get a quick overview of the config file, see an example file [here](https://github.com/bioimage-io/pytorch-bioimage-io/blob/master/specs/models/unet2d_nuclei_broad/UNet2DNucleiBroad.model.yaml).
+[This specification](./generated/bioimageio_model_spec.md) defines a file format for representing pretrained AI models in [YAML format](https://en.wikipedia.org/wiki/YAML). This format is used to describe models hosted on the [BioImage.IO](https://bioimage.io) model repository site.
```

Comment:
I think eventually we should also render this on the bioimage.io doc page and then give that as the main reference.
But we can like to the md on github for now.